### PR TITLE
Expose metric for how long have actions runners been running

### PR DESCRIPTION
**Pull request by aosoriodw**
## Description
This PR exposes a Prometheus metric like this:
```
# HELP action_runner_runtime How long has an action runner been up.
# TYPE action_runner_runtime gauge
action_runner_runtime{instance_id="i-0b0dc4854c533a6aa",label="macOS-arm64"} 38606.473834576
action_runner_runtime{instance_id="i-0c284baeba8ee0957",label="macOS-arm64"} 8745.473845792
```

This will be used in a Grafana dashboard and in an alert to notify when a runner has been up for longer than expected.
 
To be able to expose this metric, I had to add labels to runners so other EC2 instances ca be filtered out. The following labels were added:
- `app`: It's always set to `github-runner-provisioner`
- `owner`: The owner of the repository that triggered a job. For example, for `github.com/datawire/infra-actions`, owner is `datawire`
- `repo`: The repository that triggered a job. For example, for `github.com/datawire/infra-actions`, repo is `infra-action`
- `runnerLabel`: The label assigned to the GitHub runner. e.g., macOS-arm64.

## Tests
All tests were done manually. I'll send a new PR with unit tests for the code added.
1. Only instances with tag `app=github-runner-provisioner` are shown in the metrics
2. Only running instances are show in the metrics
3. Runner provisioning is not broken
4. New runners created by this code have the right labels assigned to them. See screenshot.

## Secreenshots
image

### CODE REVIEW

File: github-runner-provisioner/internal/aws/ec2.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+			break
+		}
+
+		nextToken = describeInstancesOutput.NextToken
+
+	}
+	return instancesDetails, nil
+}
+
+func getActionRunnerLabel(instance types.Instance) (string, interface{}) {
```
Comment (AidanWenzel) [inline github-runner-provisioner/internal/aws/ec2.go:63]: What is the expected result when returning an empty interface? Can it just be cast to anything?

File: github-runner-provisioner/internal/aws/runner.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/datawire/infra-actions/github-runner-provisioner/internal/utils"
+)
+
+const LabelTag = "label"
+const NameTag = "app"
+const ownerTag = "owner"
+const repoTag = "repo"
```
Comment (AidanWenzel) [inline github-runner-provisioner/internal/aws/runner.go:11]: Why are some of these camelCase and others Pascal?

**Review by AidanWenzel — APPROVED** [review]

File: github-runner-provisioner/internal/aws/ec2.go (hunk 1 above)
Comment (aosoriodw) [reply github-runner-provisioner/internal/aws/ec2.go]: @AidanWenzel that's a mistake. It should be an error, not interface. I've updated the code

File: github-runner-provisioner/internal/aws/runner.go (hunk 2 above)
Comment (aosoriodw) [reply github-runner-provisioner/internal/aws/runner.go:11]: capitalized variabels are public. Only NameTag is used outside of this package so it's the only one exposed.

**Review by AidanWenzel — APPROVED** [review]

**Pull request merged**